### DISCUSSION TITLE
Error when Init cluster inside the transient k3s-init service

### DISCUSSION THREAD

**Discussion by vitorafgomes** [techno-tim/k3s-ansible#353] [Questions] [answered]
Can anyone help me with this error that is occurring below.

TASK [k3s_server : Init cluster inside the transient k3s-init service] ***********************************
fatal: [192.168.50.215]: FAILED! => {"msg": "An unhandled exception occurred while templating '{% if groups[group_name_master | default('master')] | length > 1 %}\n  {% if ansible_hostname == hostvars[groups[group_name_master | default('master')][0]]['ansible_hostname'] %}\n    --cluster-init\n  {% else %}\n    --server https://{{ hostvars[groups[group_name_master | default('master')][0]].k3s_node_ip | split(\",\") | first | ansible.utils.ipwrap }}:6443\n  {% endif %}\n  --token {{ k3s_token }}\n{% endif %} {{ extra_server_args | default('') }}'. Error was a <class 'jinja2.exceptions.TemplateRuntimeError'>, original message: No filter named 'split' found."}
changed: [192.168.50.210]

**Comment by bluevulpine** [comment, marked as answer]
Upgrade ansible. That should fix it. 

https://stackoverflow.com/a/70870999

"The split filter wasn't added until February 2021. I think that means if you're running anything earlier than Ansible 2.11 you won't have it."

**Reply by vitorafgomes** [reply]
Thank you very much, I just updated it, and it worked.
